src/chapters/jdbc-auth.xml

```diff
@@ -118,6 +225,135 @@
+                                                </listitem>
+                                            </varlistentry>
+                                            <varlistentry>
+                                                <term>verify-full</term>
+                                                <listitem>
+                                                    <para>Require SSL connections, 
+                                                        verifying that the server certificate
+                                                        is issued by a known authority, and
+                                                        that the name on the certificate
+                                                        matches the name of the server.</para>
+                                                </listitem>
+                                            </varlistentry>
+                                        </variablelist>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-cert-file</property></entry>
+                                    <entry>
+                                        <para>The client certificate file to use when making
+                                            an SSL connection to the Postgres server. This
+                                            property is optional, and will be ignored if
+                                            the SSL mode is set to disable.</para>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-key-file</property></entry>
+                                    <entry>
+                                        <para>The file containing the client private key
+                                            to be used when making an SSL-encrypted connction
+                                            to a Postgres server.  This property is optional
+                                            and will be ignore if the SSL mode is set to
+                                            disable.</para>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-root-cert-file</property></entry>
+                                    <entry>
+                                        <para>The file that contains the root certificates
+                                            against which the server certificate verification
+                                            will be performed.  This property is optional and
+                                            is only used if the SSL mode is set to 
+                                            verify-ca or verify-full.</para>
```

Comment:
Is this property required in all cases if `verify-ca` or `verify-full` are specified? Is the default Java truststore used?

Reply:
Need to double-check this behavior...

Reply:
Okay, I have tested and documented the behavior.  It is full of nuances, so I've tried to summarize it as best I can...